### PR TITLE
TFW-4116 - Add HelperList to HelperModal

### PR DESCRIPTION
**Pull request by jasonhale**
Adds the new HelperList to the universal search modal and to demo.

Screen Shot 2019-07-26 at 12 01 51 PM

### CODE REVIEW

File: components/universalSearch/helperModal.js [hunk 1]
```diff
@@ -16,12 +16,32 @@ export class HelperModalComponent extends PureComponent {
         selections: []
     };
 
-    onHelperChange = (displayValue, value, rawValue) => {
-        this.setState({ currentSelection: { displayValue, value, rawValue } });
+    onHelperChange = (type, displayValue, value, rawValue) => {
```
Comment (tVeratti) [inline components/universalSearch/helperModal.js]: Looks like we'll have a bit on conflicting code here. Shouldn't be too difficult to sort out, though.

File: components/universalSearch/helperModal.js [hunk 2]
```diff
@@ -101,8 +118,10 @@ export class HelperModalComponent extends PureComponent {
               <div className={'helper-modal__helper'}>
                 <HelperSwitch
                   selection={rawValue}
+                  selections={selections}
                   attributes={activeDimension.attributes}
                   onChange={this.onHelperChange}
+                  onHelperValueChange={this.onHelperValueChange}
```
Comment (tVeratti) [inline components/universalSearch/helperModal.js]: Could we rename this to `onValueChange` or `onInput`? It seems a bit redundant with the Helper included :) If you disagree it's cool to keep it as-is, but then I'd suggest `onChange` should be consistent.

Comment (jasonhale) [reply components/universalSearch/helperModal.js]: `onValueChange` makes sense to me.  Will make that change.

Comment (jasonhale) [reply components/universalSearch/helperModal.js]: @tVeratti - updated

**Pull request merged**